acl_loader/main.py

```diff
@@ -160,17 +206,34 @@ def read_sessions_info(self):
         Read MIRROR_SESSION table from configuration database
         :return:
         """
-        self.sessions_db_info = self.configdb.get_table(self.CFG_MIRROR_SESSION_TABLE)
+
+        """
+        For multi-npu platforms we will read from any one of front asic namespace 
+        config db as the information should be same across all config db
+        """
+        if self.per_npu_configdb:
+            namespace_configdb = (self.per_npu_configdb.values())[0]
```

Comment:
Will the table CFG_MIRROR_SESSION_TABLE have different keys between the TABLE in global and namespace DB ?

Reply:
@judyjoseph global db will not have key on muti-npu platforms and it will same for all front asic namespace.